tests/Foundation/FoundationInteractsWithDatabaseTest.php

```diff
@@ -30,6 +30,8 @@ class FoundationInteractsWithDatabaseTest extends TestCase
     protected function setUp(): void
     {
         $this->connection = m::mock(Connection::class);
+
+        $this->connection->shouldReceive('getName')->andReturn('test_connection');
```

Comment:
Why did you need to add this to the `setUp` method? Does this PR changes the behaviour of all method calls in Eloquent?